arm/Microsoft.Authorization/policyExemptions/.parameters/mg.min.parameters.json

```diff
@@ -0,0 +1,15 @@
+{
+    "$schema": "https://schema.management.azure.com/schemas/2019-04-01/deploymentParameters.json#",
+    "contentVersion": "1.0.0.0",
+    "parameters": {
+        "name": {
+            "value": "<<namePrefix>>-min-mg-polexem"
+        },
+        "policyAssignmentId": {
+            "value": "/providers/Microsoft.Management/managementGroups/<<managementGroupId>>/providers/Microsoft.Authorization/policyAssignments/adp-<<namePrefix>>-mg-pass-loc-rg"
+        },
+        "managementGroupId": {
```

Comment:
Isn't this value automatically provided by the pipeline (given the scope)

Reply:
I've removed on the 'min' and kept on the normal one to demonstrate the use of the parameter. It is 'optional' and will use the current deployment scope (thanks to the new managementGroup() function)